marathon_lb.py

```diff
@@ -576,6 +626,9 @@ def __init__(self, appId, servicePort, healthCheck):
         self.servicePort = servicePort
         self.backends = set()
         self.hostname = None
+        self.proxypath = ''
```

Comment:
Any reason not to default to None and then check for `if app.proxypath:` instead of `if app.proxypath != '':`? It's the style used for the rest of the code. Same for revproxypath and redirpath.